packages/block-library/src/gallery/v1/edit.js

```diff
@@ -94,6 +94,18 @@ function GalleryEdit( props ) {
 		blockEditorStore
 	);
 
+	// Remove the tools panel for v1 Gallery so we only have to support
+	// any new dimension settings, etc. for the new gallery format.
+	const toolsPanel = document.getElementsByClassName(
```

Comment:
The CSS rule is working nicely to hide the ToolsPanel, and I like it as a pragmatic workaround for not being able to conditionally update the block's supports settings / alter whether or not the panel is displayed.

This might be overkill (and I'm maybe being overly cautious), but I was wondering if it's worth making the check a little more specific to the block inspector? What do you think about something like the following:

```
const toolsPanel = document.querySelector( '.block-editor-block-inspector .components-tools-panel' );
```

Reply:
Another case that we probably can't really do much about is whether or not the `gap` control is displayed in global styles for this block 🤔